### PR TITLE
Init skeleton defined with env vars (. nor source worked)

### PR DESCRIPTION
**Pull request by JorMaister**
Remote branch dh_models contains that init to test the init proposed with a protocol.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,107 @@
-# Ignore temporary packaging folder
-*.egg-info
+# Byte-compiled / optimized / DLL files
+__pycache__/
```
Comment (pconesa) [inline .gitignore]: Why so many ignore items?

File: dynamo/__init__.py [hunk 2]
```diff
@@ -24,12 +24,50 @@
...
+    @classmethod
+    def getEnviron(cls):
+        """ Create the needed environment for Dynamo programs. """
+        environ = pwutils.Environ(os.environ)
+        dyn_home = cls.getHome()
+        environ.update({
+            'PATH': dyn_home + '/matlab/bin:' + dyn_home + '/matlab/src:' + dyn_home + '/cuda/bin:' + dyn_home +
+                    '/mpi:${PATH}',
+            'LD_LIBRARY_PATH': dyn_home + '/MCRLinux/runtime/glnxa64:' + dyn_home + '/MCRLinux/bin/glnxa64:' +
+                               dyn_home + '/MCRLinux/sys/os/glnxa64:${LD_LIBRARY_PATH}',
```
Comment (pconesa) [inline dynamo/__init__.py]: Just a question, do we actually need this at the end? ${LD_LIBRARY_PATH}

File: dynamo/__init__.py [hunk 3]
```diff
@@ -24,12 +24,50 @@
...
+            'DYNAMO_ROOT': dyn_home
+        }, position=pwutils.Environ.BEGIN)
+        return environ
+
+    @classmethod
+    def getDynamoProgram(cls):
+        return join(cls.getHome(), 'matlab', 'bin', DYNAMO_PROGRAM)
+
+
```
Comment (pconesa) [inline dynamo/__init__.py]: Why don't we add a helper method "runDynamoProgram" than will:
1.- activate dynamo
2.- run dynamo with some commands

Comment (JorMaister) [reply dynamo/__init__.py]: runDynamoProgram added.

File: dynamo/__init__.py (hunk 2 above)
Comment (JorMaister) [reply dynamo/__init__.py]: As you said, it ins't necessary with the BEGIN functionality.

File: .gitignore (hunk 1 above)
Comment (JorMaister) [reply .gitignore]: I just copied it from another plugin's init, most of the folders and extensions may not appear in this plugin, I can clean it.

**Pull request merged**